### ISSUE TITLE
Can someone explain what `audience` is and how it's used

### ISSUE THREAD

**Issue by Kinbaum** [VirtusLab-Open-Source/strapi-plugin-navigation#148]
Hi, I've seen discussions here about leveraging the audience field for various customizations; including things such as grouping nav items under a non-linkable dropdowns. I can't find and information/documentation on how to actually leverage this field.

What does this actually do, what is its intended purpose, and how do I get something to show up in the JSON response?

**Comment by cyp3rius** [comment]
Hello @Kinbaum ,

first of all you must enable `audience` field in you [configuration](https://github.com/VirtusLab-Open-Source/strapi-plugin-navigation#-configuration)

After that you'll get new dropdown in the item form. To populate data to that dropdown you must add at least single audience in `Content Manager` -> `Audience`.

**Comment by Kinbaum** [comment]
That's what I thought, but I'm not getting that. There is no dropdown in the new item form. I've added an audience item.

```js
navigation: {
    enabled: true,
    config: {
        additionalFields: [ 'audience' ],
        allowedLevels: 3,
        contentTypes: [
            'api::homepage.homepage',
            'api::public-page.public-page'
        ],
        contentTypesNameFields: {
            'api::public-page.public-page': [ 'title' ]
        }
    }
}
``
<img width="832" alt="Screen Shot 2022-02-07 at 7 35 51 AM" src="https://user-images.githubusercontent.com/6590356/152789173-d04d6ba9-98eb-4074-b44b-61c6b9ba4867.png">
`

**Comment by cyp3rius** [comment]
@ltsNotMike can you look on this?

**Comment by Kinbaum** [comment]
@cyp3rius @ltsNotMike It looks like this config item is never used in this form. There is also no dropdown component associated with audience.

https://github.com/VirtusLab-Open-Source/strapi-plugin-navigation/blob/af73f18e405bbc67ecf66b394a11bb3355089b39/admin/src/pages/View/components/NavigationItemForm/index.js#L30

**Comment by cyp3rius** [comment]
That seems we missed the user end here. @ltsNotMike  it working on it now.

**Comment by cyp3rius** [comment]
Fix releases as part of `v2.0.1`
